helpdesk_mgmt_project/models/__init__.py

```diff
@@ -1,4 +1 @@
-from . import helpdesk_ticket
-from . import helpdesk_ticket_team
-from . import project
-from . import project_task
+from . import helpdesk_ticket, helpdesk_ticket_team, project, project_task
```

Comment:
strange lint here, 
I think that is not the one produced by odoo-lint, could you check ?